opensrp-reveal/src/main/java/org/smartregister/reveal/viewholder/DownloadedOfflineMapViewHolder.java

```diff
@@ -0,0 +1,54 @@
+package org.smartregister.reveal.viewholder;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.CheckBox;
+import android.widget.TextView;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.model.OfflineMapModel;
+
+public class DownloadedOfflineMapViewHolder extends RecyclerView.ViewHolder {
```

Comment:
this class like a replicate of `AvailableOfflineMapViewHolder`, both have 2 textviews and 1 checkbox. Maybe you can refactor to 1 view holder and have them extend a common viewholder